appscale/tools/templates/AppScalefile

```diff
@@ -0,0 +1,160 @@
+---
+
+
+##################################################
+########## Virtual Machine (VM) Layout ###########
+##################################################
+# The deployment strategy used in this AppScale deployment.
+# Each service/role can run on one or multiple machines.
+# Primary roles: master, appengine, database, zookeeper
+# Optional roles: loadbalancer, taskqueue, search
+
+# For each role other than master you can specify multiple machines.
+# Use the IPs of existing machines,
+# or node-X notation if you want AppScale to start new machines on the
+# Infrastructure of choice.
+# The following layout provides Fault Tolerance and High Availability:
+# ips_layout:
+#  master : [vm1_private_IP | node-1]
+#  appengine:
+#   - [vm2_private_IP | node-2]
+#   - [vm3_private_IP | node-3]
+#  database:   # Cassandra needs a quorum to answer queries (hence 3)
```

Comment:
If we have replication set to 3 by default, what would be the behavior when starting AppScale with 1 database instead of 3?

Would that make more sense since we propose 3 DBs at all times?